tensorflow/lite/toco/graph_transformations/identify_lstm.cc

```diff
@@ -202,8 +202,14 @@ ::tensorflow::Status IdentifyLstmCell::Run(Model* model, std::size_t op_index,
     return ::tensorflow::Status::OK();
   }
   // State forget gate activation function
-  if (!MatchOperatorInputs(*state_forget_sig, *model, OperatorType::kSplit,
-                           &tmp) ||
+  Operator *state_forget_add;
+  if (!MatchOperatorInputs(*state_forget_sig, *model, OperatorType::kAdd,
```

Comment:
@jyoungyun It is correct for LSTM that "Spilt" comes before "forget sigmoid" without "Add".
BTW, my LSTM model using "static_rnn" also has the "Add". It's strange.